docs/kusama-social-recovery.md

```diff
@@ -0,0 +1,143 @@
+> used for reserved when doing the recovery.
+
+First, use your new account that shows in the [Account](https://polkadot.js.org/apps/#/accounts)
+page to initiate a recovery process by clicking the "Initiate recovery for another" option that is
+in the menu besides the send button.
+
+![](/img/recovery/social-recovery-4.png)
+
+Then input the address you would like to recover in "recover this account" field and click "Start
+recovery".
+
+![](/img/recovery/social-recovery-5.png)
+
+Once the transaction went through, some KSMs will be locked to prevent malicious behavior.
+
+![](/img/recovery/social-recovery-6.png)
+
+Now call your friends that you have set in the first section. They are required to submit a
+`vouchRecovery` transaction that provides the lost and the rescuer address.
+
+![](/img/recovery/social-recovery-7.png)
+
+Once the threshold is reached and the block delay is passed, use the new account to submit a
+`claimRecovery` transaction that would set a proxy on behalf of your lost account in the underlying.
+It means that you can still use the lost account in an indirect way to interact with the network.
+
+![](/img/recovery/social-recovery-8.png)
+
+To see the proxy information, use your new account by calling the "recovery->proxy(Accountid)"
+function at the [Chain state](https://polkadot.js.org/apps/#/chainstate) page. It should point to
+your lost account.
+
+![](/img/recovery/social-recovery-9.png)
+
+Next, in order to call the "closeRecovery" transaction, you can make use of the "asRecovered" as
+your lost account to get the locked KSM.
+
+![](/img/recovery/social-recovery-10.png)
+
+Once the transaction went through, the reserved KSM from the "NEW-ACC" has been moved to the lost
+account. This is a way of preventing someone from recovers other accounts maliciously. Imagine if
+someone tries to initiate recovery on your account, you can do this to slash their locked KSMs.
+
+![](/img/recovery/social-recovery-11.png)
+
+Moving on, by using the "asRecovered" to submit the `removeRecovery` transaction on behalf of the
+lost account to release the reserved KSM from your lost account.
+
+![](/img/recovery/social-recovery-12.png)
+
+Now all your account balance should be transferable.
+
+![](/img/recovery/social-recovery-13.png)
+
+Finally, transfer all of your available balance from the lost account to the new account.
+
+![](/img/recovery/social-recovery-14.png)
+
+Congratulation! You have successfully done the recovery process.
+
+![](/img/recovery/social-recovery-15.png)
+
+There is still one possible way to recover the account without going through the recovery process.
+That is root. However, while using the root permission to perform this action, you will have to go
+through the governance by convincing the council members to help you to submit a council proposal or
+submit a public proposal yourself to see whether they will approve it or not. To learn more about
```

Comment:
Rephrase ->

In order to use root permissions you will need to either go through the council or submit a public proposal.